pnc-rest/src/main/java/org/jboss/pnc/rest/debug/TestEndpoint.java

```diff
@@ -0,0 +1,32 @@
+package org.jboss.pnc.rest.debug;
+
+import com.wordnik.swagger.annotations.Api;
+import com.wordnik.swagger.annotations.ApiOperation;
+import org.jboss.pnc.spi.events.BuildStatusChangedEvent;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.event.Event;
+import javax.inject.Inject;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+@Api(value = "/test", description = "This is special Endpoint for testing purpose. Use with caution.")
+@Path("/test")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+@ApplicationScoped
+public class TestEndpoint {
+
+    @Inject
+    Event<BuildStatusChangedEvent> buildStatusChangedEventEvent;
+
+    @POST
+    @Path("/buildstatus/notify")
+    @ApiOperation(value = "Sends BuildStatusChangedEvent just like it was from Core, useful for testing WebSockets")
+    public void sendBuildStatusChangedEvent(BuildStatusChangedEventRest buildStatusChangedEventRest) {
```

Comment:
@alexcreasy @vibe13 This is the most convenient way to test web sockets. It simulates changing build status from core.